stream_alert_cli/test.py

```diff
@@ -25,8 +25,10 @@
 import boto3
 from moto import mock_s3
 from stream_alert.handler import StreamAlert
-# import all rules loaded from the main handler
-import main
+# import all rules so they are loaded
+# pylint: disable=unused-import
+from rules import sample_rules, sample_matchers
```

Comment:
IMO importing main here is confusing as to the purpose of doing so. This also avoids the other unnecessary import that occurs by importing main. Happy to change it back if you disagree, though.